tests/unit/FirebaseObject.spec.js

```diff
@@ -480,13 +480,6 @@ describe('$firebaseObject', function() {
       expect(obj.$remove()).toBeAPromise();
     });
 
-    it('should set $value to null and remove any local keys', function() {
```

Comment:
I'm not sure I agree that this test should be removed. I'm fine getting rid of `$utils.dataKeys()`, but this test is still valid and should stick around.